tests/test_cli_sync.py

```diff
@@ -450,3 +450,16 @@ def test_allow_in_config_pip_compile_option(run, runner, tmp_path, make_config_f
 
     assert out.exit_code == 0
     assert "Using pip-tools configuration defaults found" in out.stderr
+
+
+@mock.patch("piptools.sync.run")
```

Comment:
Don't use `mock` directly. In pytest, use the `mocker` fixture or `monkeypatch` if possible.

Reply:
I've adapted the test style to the rest of the file.

Reply:
Okay, then this is out of the scope of this PR. Marking the thread as resolved.